rules/windows/defense_evasion_sdelete_like_filename_rename.toml

```diff
@@ -12,18 +11,19 @@ file overwrite and rename operations.
 """
 from = "now-9m"
 index = ["winlogbeat-*", "logs-endpoint.events.*"]
-language = "lucene"
+language = "eql"
 license = "Elastic License"
 name = "Potential Secure File Deletion via SDelete Utility"
 note = "Verify process details such as command line and hash to confirm this activity legitimacy."
 risk_score = 21
 rule_id = "5aee924b-6ceb-4633-980e-1bde8cdb40c5"
 severity = "low"
 tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
-type = "query"
+type = "eql"
+
 
 query = '''
-event.category:file AND event.type:change AND file.name:/.+A+\.AAA/
+file where event.type == "change" and wildcard(file.name,"*AAA.AAA")
```

Comment:
is the file name always uppercase? i.e., not case sensitive?

Reply:
yes